sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md

```diff
@@ -141,6 +141,19 @@ Use with [DwmSetWindowAttribute](/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindo
 
 The maximum recognized <b>DWMWINDOWATTRIBUTE</b> value, used for validation purposes.
 
+### -field DWMWA_USE_IMMERSIVE_DARK_MODE:20
+
+Use with [DwmSetWindowAttribute](/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindowattribute). Allows the window frame for this window to be drawn in dark mode colors when the dark mode system setting is enabled. For compatibility reasons, all windows default to light mode regardless of the system setting. The <i>pvAttribute</i> parameter points to a value of type **BOOL**. <b>TRUE</b> to honor dark mode for the window, <b>FALSE</b> to always use light mode.
+
+This value is supported starting with Windows 11 Build 22000.
```

Comment:
```suggestion
<b>Windows 10 before build 10.0.17763: </b>This value is not supported.

<b>Windows 10 build 10.0.17763 until 10.0.18985: </b>This value is 19 instead.

<b>Windows 10 build 10.0.18985 or later: </b>This value is supported.
```

Reply:
@AraHaan  - Please do not update this pull request. This is a test to try to diagnose some issues in our build system. There is an update to these docs coming in the next day or two that has been driven and reviewed by the feature owners. I'm sorry that there was confusion around your updates, but we will not be completing this PR.

Reply:
Thanks for explaining it.

Reply:
@AraHaan I have a little more info about this. It turns out that the API was added to the header, but was not actually supported in the OS until Windows Build 22000, and that's why your initial documentation was reverted at the time.

Reply:
Also I m guessing that by the part of the docs that says to pass ``TRUE`` would mean that it won't use dark mode if the user configured the system to use light mode. However what if my application code requires to override that option via this method to use dark mode regardless of the system setting? Unless there are plans for that eventually as well (without people like me being forced to custom draw the frame that is).